### PR TITLE
update 2022 to 2023

### PR DESCRIPTION
**Pull request by Kriesse**
update logos and outdated bits of copy.

image

### CODE REVIEW

File: _includes/join-us-de.html [hunk 1]
```diff
@@ -5,18 +5,16 @@
...
-          unten, um neue Arbeitsumgebungen auszuprobieren und zu erleben, was Coworking bedeutet.
+          Hear first about all news around Coworking Festival Berlin & Brandenburg.
         </p>
-
-        <p class="tc mt4">
-          <a href="https://onecoworking.com/coworking-festival-2022" class="btn btn--big btn--border-with-shadow">
-            Jetzt Gutscheine sichern
+        <p class="tc mt4 flex gap3">
+          <a href="https://www.linkedin.com/company/berlincoworking/" class="btn btn--big btn--border-with-shadow">
+            Linkedin
```
Comment (SubandiK) [inline _includes/join-us-de.html:14]:
```suggestion
            LinkedIn
```

File: _includes/join-us.html [hunk 2]
```diff
@@ -5,17 +5,16 @@
...
-          check in with a simple click. Register below to try out new workspaces and experience what coworking is all
-          about.
+          Hear first about all news around Coworking Festival Berlin & Brandenburg.
         </p>
-        <p class="tc mt4">
-          <a class="btn btn--big btn--border-with-shadow" href="https://onecoworking.com/coworking-festival-2022">
-            Get Free Vouchers
+        <p class="tc mt4 flex gap3">
+          <a href="https://www.linkedin.com/company/berlincoworking/" class="btn btn--big btn--border-with-shadow">
+            Linkedin
```
Comment (SubandiK) [inline _includes/join-us.html:14]:
```suggestion
            LinkedIn
```

File: _includes/join-us-de.html [hunk 3]
```diff
@@ -5,18 +5,16 @@
     </div>
     <div class="media-content__content">
       <div class="border-with-shadow pa4 pa5-l">
-        <h3>Gratis Coworking mit der One Coworking App</h3>
+        <h3>Stay connected</h3>
```
Comment (thilo) [inline _includes/join-us-de.html:8]: Das ist für die Deutsch Version.

File: de/tours.html
Comment (thilo) [inline de/tours.html:1]: If unlick from the menu no need to change that. Than you don't need to update the english and german version.

**Review by SubandiK — APPROVED** [review]
two times same tiny "typo": "Linkedin" instead of "LinkedIn" as they style themselves

File: en/schedule.html
Comment (thilo) [inline en/schedule.html:1]: If unlick from the menu no need to change that. Than you don't need to update the english and german version.

File: index.html
Comment (thilo) [inline index.html:1]: index-de. needs same update.

**Review by carmem-s — APPROVED** [review]

**Pull request merged**